app/components/icons/ArrowFilterIcon.tsx

```diff
@@ -3,30 +3,13 @@ import { useState } from "react";
 export default function ArrowFilterIcon({ classNames = "h-6 w-6" }) {
   const [isActive, setIsActive] = useState(false);
 
-  const [isHover, setIsHover] = useState(false);
-
-  const handleMouseEnter = () => {
-    setIsHover(true);
-  };
-
-  const handleMouseLeave = () => {
-    setIsHover(false);
-  };
-
-  const arrowStyle = {
-    color: isHover ? "#9cc317" : "",
-  };
-
   return (
     <>
       <div className="accordion" style={{ margin: "0 1rem" }}>
```

Comment:
As mentioned further down for the autogenerated file, the `accordion` class needs to be replaced. And as mentioned in another comment, the `style` can also be replaced with `mx-4` :)